packages/fetchai/skills/simple_oracle/skill.yaml

```
@@ -0,0 +1,62 @@
+name: simple_oracle
+author: fetchai
+version: 0.1.0
+type: skill
+description: This skill deploys a Fetch oracle contract
+license: Apache-2.0
+aea_version: '>=0.7.0, <0.8.0'
+fingerprint:
+  __init__.py: QmNkZAetyctaZCUf6ACxP5onGWsSxu2hjSNoFmJ3ta6Lta
+  behaviours.py: QmT1qe1BanTBU9TCzJc3ZPSzcDMMD9qK8wnjK1F4hy6SoC
+  dialogues.py: QmfYuTaa2zs9Cuyg6T8tbbEgkQNnPFhaAgyxy8oF8YUqPv
+  handlers.py: QmTT95chwza2p68DmdkLfrJcZ5ZrAu9rRTSM5hd3zTTj15
+  strategy.py: QmSv2KUKeuhVZJfvA9TbZGZtRSMbWrnVGawNiSxv9cWBah
+fingerprint_ignore_patterns: []
+contracts:
+- fetchai/oracle:0.1.0
+protocols:
+- fetchai/contract_api:0.8.0
+- fetchai/default:0.9.0
+- fetchai/ledger_api:0.7.0
+- fetchai/signing:0.7.0
+skills: []
+behaviours:
+  oracleer:
+    args:
+      update_interval: 15
+    class_name: FetchOracleDeployer
+handlers:
+  contract_api:
+    args: {}
+    class_name: ContractApiHandler
+  ledger_api:
+    args: {}
+    class_name: LedgerApiHandler
+  signing:
+    args: {}
+    class_name: SigningHandler
+models:
+  contract_api_dialogues:
+    args: {}
+    class_name: ContractApiDialogues
+  ledger_api_dialogues:
+    args: {}
+    class_name: LedgerApiDialogues
+  signing_dialogues:
+    args: {}
+    class_name: SigningDialogues
+  strategy:
+    args:
+      contract_address: null
+      default_gas_deploy: 100000
+      default_gas_grant_role: 100000
+      default_gas_update: 150000
+      erc20_address: '0x0000000000000000000000000000000000000000'
```

what is the meaning of this? Why `0x00...0`?

---

Good question, maybe I can add a comment. The reason is that there is an explicit check for this in the contract. This value indicates that the payment for the oracle value will be in ETH, while a valid FET ERC20 contract address means the payment will be in FET.

---

Thanks for the explanation, makes sense. 
> The reason is that there is an explicit check for this value in the contract.
We could have a check for `null`, `"null"`, `None` or equivalent, assuming Parity has an analogous type. That would make this part self-documenting. However, I don't think it's a priority, just a thought I had.

You could maybe add a comment just above that line, or at the end of the line, something like:
```suggestion
      # address of the ERC20 token contract to use for payment to the oracle.
      # Default: use the native currency ETH.
      erc20_address: '0x0000000000000000000000000000000000000000'
```